src/provider/index.ts

```diff
@@ -168,18 +151,29 @@ function decodeID(payload: Notification & SenderInfo | NotificationEvent | {id:
  * @param {Object} payload The contents to be dispatched to the UI
  * @param {Object} sender Window info for the sending client. This can be found in the relevant app.json within the demo folder.
  */
-async function createNotification(payload: Notification, sender: ProviderIdentity) {
+async function createNotification(payload: CreatePayload, sender: ProviderIdentity): Promise<Notification> {
     // For testing/display purposes
     console.log('createNotification hit');
 
     console.log('payload', payload);
     console.log('sender', sender);
 
-    testDisplay('createNotification', payload, sender);
+    const notification: Notification = {
+        id: payload.id,
+        body: payload.body,
+        title: payload.title,
+        subtitle: payload.subtitle || '',
+        icon: payload.icon || '',
```

Comment:
Do we need to set values when the `Notification` interface allows these properties to be undefined?